mmvb_backend/common/definitions.py

```diff
@@ -1,5 +1,13 @@
 import json
 import os
+from enum import Enum
+
+
+class HealthCheckStatus(Enum):
```

Comment:
i just wanted to raise to your awareness that i defined something similar here: [benchmarking_sessions/models.py:16](https://github.com/FG-AI4H-TG-Symptom/fgai4h-tg-symptom-assessment-mmvb-backend/blob/develop/mmvb_backend/benchmarking_sessions/models.py#L16)
i don't want to say it necessarily needs to be merged, just so that you're aware